lib/src/theme/alarm_colors_extension.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:hmi_core/hmi_core_alarm_colors.dart';
+/// 
+/// Extention to add [AlarmColors] to [ThemeData]
```

Comment:
Add description to the classes
1 - fatal fault, loss of functionality
3 or 4 - ...

Reply:
- Added description and usage info for `AlarmColorsExtension`
- Described colors in `AlarmColors` docs